### ISSUE TITLE
[Bug] Performance problems with kvantum on Plasma 5.22.3

### ISSUE THREAD

**Issue by eredrolo** [tsujan/Kvantum#672]
When I use a Kvantum theme like Layan, the fps of the desktop drops to 30 when I maximise a window which is using kvantum (like Dolphin, Kate, qBittorrent). Turning off kvantum fixes this.

This started when I updated to Plasma 5.22 and when I use Intel UHD 630 GPU. When I use my GTX 1060 there is no fps drop.

**System Information:**  
Kvantum version: 0.20.1
Operating System: Manjaro Linux  
KDE Plasma Version: 5.22.3  
KDE Frameworks Version: 5.84.0  
Qt Version: 5.15.2  
Kernel Version: 5.10.49-1-MANJARO (64-bit)  
Graphics Platform: X11  
Processors: 4 × Intel® Core™ i3-8100 CPU @ 3.60GHz  
Memory: 15.5 GiB of RAM  
Graphics Processor: Mesa Intel® UHD Graphics 630  
Mesa version: 21.1.4-1

**Comment by tsujan** [comment]
Kvantum can't and doesn't affect fps. Probably you use a translucent theme with blurring. You may want to check Plasma's compositor settings and/or your driver settings.

**Comment by tsujan** [comment]
> When I use a Kvantum theme like Layan

I took a look at Layan's screenshots: Yes, it's a translucent theme with blurring. Either your GPU can't fully support it or (if it did before the upgrade) something has gone wrong in compositor's settings.

**Comment by Dkoeddnsnp** [comment]
Same issue on my system with AMDGPU and Plasma 5.22.. Happend with blur on and off.

**Comment by eredrolo** [comment]
Looks like turning off blur fixes this for me. I've been using this configuration for half an year but never noticed the fps drop before. I tried changing the compositor settings but it didn't work.

**Comment by tsujan** [comment]
I also use KWin's blur effect with a translucent Kvantum theme but haven't encountered a problem after upgrading to 5.22.3. It's possible that some KDE settings have changed for some graphic cards, or maybe it's about a new bug.

However, this is outside Kvantum's territory. Kvantum doesn't blur anything; KDE does.

**Comment by eredrolo** [comment]
Thanks for the help. I'll update this if I find a fix.

**Comment by tsujan** [comment]
This may be related or a different Plasma problem:

I use KWin outside KDE (under LXQt) but, after recent upgrades, I started too see random issues in tooltips. Luckily, I was able to avoid them by disabling tooltip effects in KDE settings but they'd never happened before those upgrades.

On a positive side, the DND pixmap lagging that I'd tolerated for a long time (perhaps https://bugs.kde.org/show_bug.cgi?id=352019) suddenly disappeared.

So, I think that KDE has recently had important changes regarding graphics, with both positive and negative effects.
